src/libs/fileDownload/FileUtils.js

```diff
@@ -131,4 +131,35 @@ function appendTimeToFileName(fileName) {
     return newFileName;
 }
 
-export {showGeneralErrorAlert, showSuccessAlert, showPermissionErrorAlert, splitExtensionFromFileName, getAttachmentName, getFileType, cleanFileName, appendTimeToFileName};
+/**
+ * Reads a locally uploaded file
+ *
+ * @param {String} path - the blob url of the locally uplodaded file
+ * @param {String} fileName
+ * @returns {Promise}
+ */
+const readFileAsync = (path, fileName) =>
+    new Promise((resolve) => {
+        if (!path) {
+            resolve();
+        }
+
+        return fetch(path)
+            .then((res) => {
+                if (!res.ok) {
+                    throw Error(res.statusText);
+                }
+                return res.blob();
+            })
+            .then((blob) => {
+                const file = new File([blob], cleanFileName(fileName));
+                file.source = path;
+                resolve(file);
+            })
+            .catch((e) => {
+                console.debug('[FileUtils] Could not read uploaded file', e);
+                resolve();
```

Comment:
We are resolving even on failures? Shouldn't we reject?

Reply:
We resolve with an undefined receipt file and the call to RequestMoney proceeds without the receipt. 

I agree that we should eventually reject and display some error to the user, but I didn't see anything about it in the doc. Since this PR is blocking others in this project and we currently don't display errors in MoneyRequestConfirmPage, I think we should keep the scope of this PR smaller and tackle displaying an error as a follow up once we align on what error will be displayed.